angular/src/components/icon.component.ts

```diff
@@ -104,4 +106,16 @@ export class IconComponent implements OnChanges {
             this.image = null;
         }
     }
+
+
+    private setCardIcon() {
+        if (this.imageEnabled) {
```

Comment:
I don't believe we need this check here, it's used in `setLoginIcon` to avoid calling our icon service for those that don't wish it.